docs/src/tutorial.jl

```diff
@@ -0,0 +1,309 @@
+
+# You can use alternative algorithms in [`basins_fractions`](@ref), see
+# the documentation of [`AttractorMapper`](@ref) for possible subtypes.
+# [`AttractorMapper`](@ref) defines an extendable interface and can be enriched
+# with other methods in the future!
+
+# ## Continuation
+
+# If you have heard before the word "continuation", then you are likely aware of the **traditional continuation-based bifurcation analysis (CBA)** offered by many software, such as AUTO, MatCont, and in Julia [BifurcationKit.jl](https://github.com/bifurcationkit/BifurcationKit.jl). Here we offer a completely different kind of continuation called **attractors & basins continuation**.
+
+# A direct comparison of the two approaches is not truly possible, because they do different things.
+# The traditional linearized continuation analysis continues the curves of individual fixed
+# points across the joint state-parameter space. The attractor and basins continuation first
+# finds all attractors at all parameter values and then _matches_ appropriately similar
+# attractors across different parameters, giving the illusion of continuing them individually.
+
+# This is a fundamental difference. With our approach, one finds all attractors
+# (or almost all, for insufficiently dense sampling). And because all attractors are simultaneously
+# tracked across the parameter axis, the user may arbitrarily estimate _any_
+# property of the attractors and how it varies as the parameter varies.
+# A more detailed comparison between these two approaches can be found in [Datseris2023](@cite).
+
+# To perform the continuation is extremely simple. First, we decide what parameter,
+# and what range, to continue over:
+
+prange = 4.5:0.02:6
+pidx = 1 # index of the parameter
+
+# Then, we may call the [`continuation`](@ref) function.
+# We have to provide a continuation algorithm, which itself references an [`AttractorMapper`](@ref).
+# In this example we will re-use the `mapper` to create a [`RecurrencesFindAndMatch`](@ref) continuation algorithm.
+# This algorithm uses the `mapper` to find all attractors at each parameter value.
+# Then, it performs a "matching" step, ensuring a "continuity" of the attractor
+# label across the parameter axis. You can read the docstring for more details,
+# as this algorithm is quite sophisticated!
+
+# For now we can use all of its default options which are reliable 99% of the time
+
+rafm = RecurrencesFindAndMatch(mapper)
+
+# and call
+
+fractions_curves, attractors_info = continuation(
+	rafm, prange, pidx, sampler; samples_per_parameter = 1_000
+)
+
+attractors_info
+
+# the output is given as two vectors. Each vector is a dictionary
+# mapping attractor IDs to ther fractions, or their state space sets, respectively.
+# Both vectors have the same size as the parameter range.
+# For example, the attractors at the 34-th parameter value are:
+
+attractors_info[34]
```

Comment:
Explicit the parameter value as well in order to show that the vectors are linked
```suggestion
attractors_info[34], prange[34]
```